src/controllers/update/beneficial.owner.type.controller.ts

```diff
@@ -121,6 +121,8 @@ const getNextPage = (beneficialOwnerTypeChoices: BeneficialOwnerTypeChoice | Man
         return config.UPDATE_MANAGING_OFFICER_CORPORATE_URL;
       case ManagingOfficerTypeChoice.individual:
         return config.UPDATE_MANAGING_OFFICER_URL;
+      case BeneficialOwnerTypeChoice.relevantPeriodOtherLegal:
+        return config.UPDATE_BENEFICIAL_OWNER_OTHER_URL + "?relevant-period=true";
```

Comment:
I was thinking maybe we should put `"relevant-period=true"` inside a config constant, then we can reuse wherever we need to